### DISCUSSION TITLE
I want to use leptos_reactive with multiple runtimes on multiple green threads (tokio)

### DISCUSSION THREAD

**Discussion by ap32** [leptos-rs/leptos#3082] [Q&A] [answered]
I like signals and I want to use leptos_reactive for my pet project, but I don't know if I can use it with tokio green threads. I thought about creating multiple runtimes, but i know nothing about how functions detect which runtime is in use, so i'll better ask.

**Comment by gbj** [comment, marked as answer]
This is not supported at all with `leptos_reactive` (in leptos 0.6), and is very easy to do with `reactive_graph` which is its replacement in 0.7.

**Reply by ap32** [reply]
Thank you, I assume it's stable enough for my small project.
